test/spec_multipart.rb

```diff
@@ -42,6 +42,13 @@ def multipart_file(name)
     }.must_raise Rack::Multipart::Error
   end
 
+  it "raises a bad request exception if no body is given" do
```

Comment:
Probably want a more complete description, such as `... if no body is given but content type indicates a multipart body`

Reply:
Done, thanks for the clear test title.